features/bootstrap/FeatureContext.php

```diff
@@ -207,7 +207,7 @@ public function thereIsDummyObjectsWithDummyDateAndRelatedDummy($nb)
     public function thereIsDummyObjectsWithDummyPrice($nb)
     {
         $descriptions = ['Smart dummy.', 'Not so smart dummy.'];
-        $prices = [9.99, 12.99, 15.99, 19.99];
+        $prices = ['9.99', '12.99', '15.99', '19.99'];
```

Comment:
http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#decimal

`string` is correct.